### PR TITLE
Update Traditional_Chinese.properties

### PR DESCRIPTION
**Pull request by hhpy123**

### CODE REVIEW

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 1]
```diff
@@ -1229,26 +1229,26 @@ Lock = 鎖定
...
-Please enter a new name for your city = 
+Please enter a new name for your city = 請為您的城市輸入一個新名稱
  # Requires translation!
 Please select a tile for this building's [improvement] = 
 
 # Ask for text or numbers popup UI
 
  # Requires translation!
-Invalid input! Please enter a different string. = 
+Invalid input! Please enter a different string. = 輸入無效！請輸入不同的字符串。
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:1249]: 有點翻譯腔。We don’t say “字串” in normal speaking, don’t we? Instead we can say “名字”in this sentence.

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 2]
```diff
@@ -4157,32 +4157,32 @@ Fascism = 法西斯
...
-Communism = 
+Communism = 共產主義
  # Requires translation!
 [amount]% [stat] when constructing [unitType] buildings [cityFilter] = 
  # Requires translation!
-Order Complete = 
+Order Complete = 命令政策完善
  # Requires translation!
-Order = 
+Order = 命令
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:4185]: Looks weird. I don't remember how civ V itself translate this word, but I suppose ‘秩序' is better.

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 3]
```diff
@@ -4319,9 +4319,9 @@ Engineer = 大工程師
 Agriculture = 農業
 Starting tech = 初始科技
 
-'Shall the clay say to him that fashioneth it, what makest thou?' - Bible Isaiah 45:9 = “泥土豈可對陶工說：你在做什麼呢？”——聖經·以賽亞書 45:9
+'Shall the clay say to him that fashioneth it, what makest thou?' - Bible Isaiah 45:9 = “泥土豈可對摶弄他的說：你做什麼呢？”——聖經·以賽亞書 45:9
 Pottery = 制陶
-'Thou shalt not muzzle the ox when he treadeth out the corn.' - Bible Deuteronomy 25:4 = “牛在踹穀的時候，不可籠住它的嘴。”——聖經·申命記　25:4
+'Thou shalt not muzzle the ox when he treadeth out the corn.' - Bible Deuteronomy 25:4 = “牛在場上踹穀的時候，不可籠住他的嘴。”——聖經·申命記　25:4
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:4324]: Like these 2 changes. Gonna check Simplified_Chinese translation.:D

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 4]
```diff
@@ -4777,23 +4777,23 @@ Porcelain =
 #################### Lines from UnitPromotions from Civ V - Vanilla ####################
 
  # Requires translation!
-Sword = 
+Sword = 劍
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:4780]: Just for warning, you always find sword before the word 'unit'. Is 劍单位 good enough? I don't think so. How about '武装' or ’带剑'(They are not actually good enough, but it should be at least an adj.).

**Review by heipizhu4 — CHANGES_REQUESTED** [review]
I suppose you should be editing in your own fork, instead of pulling a new one. You enter here: 
image
Good enough for a beginner, but still need improvements.

**Comment by hhpy123** [comment]
Thank you for some of your reminders, because I can only see the word when
translating, and I can't see where the word is used, so sometimes the
meaning is a bit strange, thank you again for your suggestion😀😀

heipizhu4 ***@***.***> 於 2022年5月23日 週一 下午8:43 寫道：

**Comment by heipizhu4** [comment]
> Thank you for some of your reminders, because I can only see the word when translating, and I can't see where the word is used, so sometimes the meaning is a bit strange, thank you again for your suggestion😀😀 
You're welcome! Next time turn on the game while translating so that you can make less mistakes! Anyway, anyone can make mistakes so don't be afraid of making mistakes. I 'll be able to check these translations (most of the times) so these errors can be ticked out.😀
Looking forward to your changes!

File: android/assets/jsons/translations/Traditional_Chinese.properties (hunk 1 above)
Comment (hhpy123) [reply android/assets/jsons/translations/Traditional_Chinese.properties:1249]: copy that

File: android/assets/jsons/translations/Traditional_Chinese.properties (hunk 4 above)
Comment (hhpy123) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4780]: can i ask what is this word use for?

Comment (hhpy123) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4780]: I could translate more precisely if I knew :)

Comment (heipizhu4) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4780]:
```
		"name": "Warrior",
		"unitType": "Sword",
		"movement": 2,
		"strength": 8,
		"cost": 40,
		"obsoleteTech": "Metal Casting",
		"upgradesTo": "Swordsman",
		"uniques" : ["May upgrade to [Spearman] through ruins-like effects"],
		"attackSound": "nonmetalhit",
        "civilopediaText": [
			{"text": "This is your basic, club-swinging fighter."}
		]
```
You can find all the units that are `sword` [here](https://github.com/yairm210/Unciv/blob/master/android/assets/jsons/Civ%20V%20-%20Vanilla/Units.json) with a quick search of `sword`.

In game, you can find 'sword' mostly in all kinds of `uniques`, basically the functions of the units/nations/etc.
In game, you can find `uniques` in the civilopedia, but unluckily I didn't remember exactly which unique used Sword :(. Just take this screenshot as a reference.
image
And that `mounted` can be replace with `sword` or other unit types in mods. You can find all of uniques that may contain `sword` by searching ‘Unitfilters’ in the [wiki](https://yairm210.github.io/Unciv/Modders/uniques/)

(PS: I'm still having a hard time differciate ‘Melee' and ’Sword‘ I'll come back later)

Comment (hhpy123) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4780]: If only by the literal meaning to distinguish
Melee and sword are completely different meanings
Melee means more similar to fighting in close quarters
The sword is just a metaphor for the sword, nothing else

**Review by heipizhu4 — CHANGES_REQUESTED** [review]
More things to correct. You can actually take Simplified_Chinese translation as a reference:D. But never copy them too often, as CHS and CHT have a slightly different language pattern.

**Comment by heipizhu4** [comment]
I have a feeling we are getting there. I hope I can't find more errors after the next commit.

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 5]
```diff
@@ -4777,23 +4777,23 @@ Porcelain =
 #################### Lines from UnitPromotions from Civ V - Vanilla ####################
 
  # Requires translation!
-Sword = 
-Mounted = 騎乘
-Siege = 攻城
+Sword = 劍士
+Mounted = 騎士
+Siege = 大力士
```
[the remark this answers is not in the document]
Comment (hhpy123) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4782]: I check line 6782 and i find noting, do you sure is line 6782 ?

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 6]
```diff
@@ -7214,32 +7214,32 @@ Global =
...
 Improvement = 
  # Requires translation!
-Resource = 
+Resource = 資源
  # Requires translation!
-Ruins = 
+Ruins = 廢墟
  # Requires translation!
-CityState = 
+CityState = 市
```
[the remark this answers is not in the document]
Comment (hhpy123) [reply android/assets/jsons/translations/Traditional_Chinese.properties:7241]: tks you suggest

File: android/assets/jsons/translations/Traditional_Chinese.properties (hunk 5 above)
[the remark this answers is not in the document]
Comment (heipizhu4) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4782]: Sorry, line 6826. I went to the wrong file.

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 7]
```diff
@@ -7214,32 +7214,32 @@ Global =
...
  # Requires translation!
-Policy = 
+Policy = 政策
  # Requires translation!
 FounderBelief = 
  # Requires translation!
 FollowerBelief = 
  # Requires translation!
-Building = 
+Building = 建造中
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:7227]:
```suggestion
Building = 建筑
```
I wonder why I missed this mistake. The `building` should be easy to find in game, and you can find them in the cityscreen.

**Comment by heipizhu4** [comment]
@yairm210 Would you mind bringing this PR into check? It is almost all done.
PS: Will #6920 #6921 #6924 conflict with each other?

**Review by heipizhu4 — APPROVED** [review]
Nice change! ❤

**Comment by hhpy123** [comment]
how do i do that

**Comment by hhpy123** [comment]
> Can you please resolve the conflicts? :)

how do i do that

**Pull request merged**